templates/routes.stub

```diff
@@ -1,14 +1,14 @@
-Route::get('generator_builder', '\InfyOm\GeneratorBuilder\Controllers\GeneratorBuilderController@builder');
+Route::get('generator_builder', '\InfyOm\GeneratorBuilder\Controllers\GeneratorBuilderController@builder')->name('io_generator_builder');
 
```

Comment:
@nikhilbhatia22  thank you for Contribution in this package.

but i think we don't need  to use `io` prefix everywhere.
Can you please remove `io` prefix from name routes.